ludwig/schema/utils.py

```diff
@@ -152,6 +173,29 @@ def Boolean(default: bool, description=""):
     )
 
 
+def Integer(default: Union[None, int] = None, allow_none=False, description=""):
```

Comment:
Reminder for myself to refactor this, `PositiveInteger`, `NonNegativeInteger` to have optional validation arguments.

(PR is pretty big already so no need to do this here)